unicorecmsgem/static/css/style.css

```diff
@@ -274,3 +274,8 @@ margin: 0 auto;
 #sajan {
   color: #4671F8;
 }
+
+#sajan_logo {
```

Comment:
I was thinking it might be better to add the height spec to the #sajan block above. That would increase the link `<a href"">` surface area, rather than just adding more white space around it